README.md

```diff
@@ -1,9 +1,54 @@
-# pm-idm
+# IDM - Identity Manager
 
-Project Management for the IDM Project
+The Identity Manager - IDM for short - is a unified digital wallet based on open-standards that aims to support multiple decentralized identities.
 
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
```

Comment:
This should be a list or add 2 empty spaces in the end to make it a new line.
Also, you may add the "Specification (wip)" pointing to https://cryptpad.fr/code/#/2/code/view/x6emT-0hXpN4eg-9wZEHaKsmgOEU+jdcyBx9-pG6nQQ/